pkg/asset/manifests/openstack/cloudproviderconfig_test.go

```diff
@@ -64,31 +64,33 @@ region      = my_region
 }
 
 func TestCloudProviderConfigSecretQuoting(t *testing.T) {
-	cloud := clientconfig.Cloud{
-		AuthInfo: &clientconfig.AuthInfo{
-			Username:   "my_user",
-			Password:   "special \\r \\n \" \\ ",
-			AuthURL:    "https://my_auth_url.com/v3/",
-			ProjectID:  "f12f928576ae4d21bdb984da5dd1d3bf",
-			DomainID:   "default",
-			DomainName: "Default",
-		},
-		RegionName: "my_region",
+	passwords := map[string]string{
+		"regular":           "regular",
+		"with\\n":           "with\\\\n",
+		"with#":             "`with#`",
+		"with$":             "with$",
+		"with;":             "`with;`",
+		"with \r \n \" \\ ": "with \\r \\n \\\" \\\\ ",
+		"with!":             "with!",
+		"with?":             "with?",
+		"with`":             "\"\"\"with`\"\"\"",
```

Comment:
This one seems bizarre. So the ini conversion wraps the whole string with triple quotes if it uses backticks? *ugh*

Reply:
Oops, didn't realise this was the 4.2 backport.